sdk/metricsadvisor/ai-metrics-advisor/samples/v1/javascript/seriesData.js

```diff
@@ -78,7 +79,7 @@ async function getMetricSeriesData(client, metricId) {
 
     for (const series of result) {
       console.log(series.definition);
-      if (series.timestamps && series.timestamps.length > 0)
+      if (series.timestamps?.length)
```

Comment:
This (`?.`) will not work unless very latest node version. Need to rewrite ts code to have `series.timestamps && series.timestamps.length > 0`

Reply:
We're okay with this going forward. We used to not allow this, but going forward we want to be able to use the newest JS features. We might add a build step to use babel to convert them, but in general we want to model modern development patterns.